articles/install-guide/index.md

```diff
@@ -23,6 +23,7 @@ In addition, the QDK provides Q# support for Jupyter Notebooks with the IQ# Jupy
 
 The QDK is available for multiple development environments. Select your preferred setup from the sections below:
 
+- [Q# standalone executable]() choose this environment if you want to run Q# on its own without a driver and using a host program like the below options.
 - [Install Q# for C#:](xref:microsoft.quantum.install.cs) choose this environment if you want to combine C# and Q# to create a C# host program that calls Q# operations.
 - [Install Q# for Python:](xref:microsoft.quantum.install.python) choose this environment if you want to combine Python and Q# to create a Python host program that calls Q# operations.
 - [Install Q# for Jupyter Notebooks:](xref:microsoft.quantum.install.jupyter) choose this environment to execute Q# code in cells with embedded text or create quantum computing interactive tutorials. Do not choose this environment if you want to combine Q# with an external classical host program.
```

Comment:
Especially in the context of emphasizing standalone usage of Q#, this feels really negative. Both standalone executables and IQ# are great features for putting the emphasis back on Q# rather than on the classical host program, such that I think we should phrase this in a positive way rather than making statements like "do not choose this environment."